model.go

```diff
@@ -9,19 +9,17 @@ import (
 )
 
 const (
-	// KindBehavior is the default kind
-	KindBehavior = "Behavior"
-
-	// KindTemplate is the template kind
-	KindTemplate = "Template"
+	KindBehavior         = "Behavior"
+	KindAbstractBehavior = "AbstractBehavior"
+	KindTemplate         = "Template"
 )
 
 // Mock represents a mock struct
 type Mock struct {
 	// Common fields
-	Kind    string `yaml:"kind,omitempty"`
-	Key     string `yaml:"key,omitempty"`
-	Include string `yaml:"include,omitempty"`
+	Kind   string `yaml:"kind,omitempty"`
+	Key    string `yaml:"key,omitempty"`
+	Extend string `yaml:"extend,omitempty"`
```

Comment:
let's hope this breaking change won't affect anything :)

Reply:
It was a secret feature (;